### ISSUE TITLE
Breaks build when running in npm test

### ISSUE THREAD

**Issue by pepelsbey** [hcodes/yaspeller#87]
## Steps to reproduce

1. `git clone git@github.com:web-standards-ru/podcast.git && cd podcast`
2. `npm install && npm test`

See [package.json](https://github.com/web-standards-ru/podcast/blob/master/package.json#L4): `npm test` is just `yaspeller episodes`.

```bash
$ node -v
v8.9.0 or v9.0.0 (the same results)
$ npm -v
5.5.1
```

## Expected behaviour

No errors or maybe just a few in the latest `episode-93.md`

## Actual behaviour

Random suggestions and syntax errors

---

But if you go with global installaton instead it works fine: `yaspeller episodes` generates no errors. A week ago it was tottaly fine: no errors. It’s probably related to `node` or `npm` version. Partial log:

```bash
> yaspeller episodes

✓ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-02.md 353 ms
✗ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-03.md 312 ms
-----
Typos: 1
1. npm (suggest: npc)
-----

SyntaxError: Unexpected token R in JSON at position 0
✓ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-04.md 408 ms
SyntaxError: Unexpected token R in JSON at position 0
✓ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-07.md 601 ms
SyntaxError: Unexpected token < in JSON at position 0
✓ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-08.md 486 ms
✓ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-09.md 567 ms
SyntaxError: Unexpected token R in JSON at position 0
✗ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-11.md 490 ms
-----
Typos: 1
1. CSSO (suggest: CSO)
-----

✓ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-12.md 537 ms
✓ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-14.md 507 ms
✗ /Users/pepelsbey/Documents/Веб-стандарты/Подкаст/Расшифровки/episodes/episode-13.md 920 ms
-----
Typos: 3
1. 55 (53:19, suggest: (55)
2. CSSX (count: 2, suggest: CSS)
3. Stylelint (suggest: Style line)
-----
```

**Comment by hcodes** [comment]
```
npm up
```
[yaspeller v4.0.2](https://github.com/hcodes/yaspeller/releases)

**Comment by pepelsbey** [comment]
Thanks! But now I get funny _No errors, Test failed_

```
…
✓ episode-N.txt 107 ms
No errors.
Checking finished: 10.082 sec.
npm ERR! Test failed.  See above for more details.
```

**Comment by hcodes** [comment]
Fixed, v4.0.3.

![podcast fish users seles podcast -fish 101x28 2017-11-07 14-03-35](https://user-images.githubusercontent.com/157520/32490565-a2a7448c-c3c4-11e7-99a6-54234f51655b.png)

**Comment by pepelsbey** [comment]
Thank you for a quick fix! You rock 🤘
